core/src/main/scala/com/salesforce/op/stages/impl/tuning/OpValidator.scala

```diff
@@ -225,6 +225,16 @@ private[op] trait OpValidator[M <: Model[_], E <: Estimator[_]] extends Serializ
     datasetsByClass.map(_.toDF().rdd)
   }
 
+  protected def prepareWithoutDag(
+    splitter: Option[Splitter],
+    training: DataFrame,
+    validation: DataFrame
+  ): (DataFrame, DataFrame) = {
+    val trainingValidation = Vector(training, validation).flatMap(df =>
```

Comment:
`val Array(prepTraining, prepValidation) = Array(training, validation)...`

Reply:
@tovbinm Are there any guidelines for coding style that should be used when contributing to Transmogrify describing details like this one?

Reply:
Nope. We currently rely on scalastyle. I was considering to add a scalafmt to auto format the code, but I struggled with creating a good style. We might use this tool to autogen a config for us? https://github.com/tanishiking/scalaunfmt